maven/apichanges.xml

```diff
@@ -142,26 +157,29 @@ is the proper place.
 
                 SEE projects/projectuiapi/apichanges.xml
 
--->
-    <head>
-      <title>Change History for the Maven Project API</title>
-      <link rel="stylesheet" href="prose.css" type="text/css"/>
-    </head>
-    <body>
+        -->
```

Comment:
These changes aren't strictly necessary, right? On the other hand they at least look harmless.